python/lsst/ts/observing/utilities/auxtel/latiss/getters.py

```diff
@@ -0,0 +1,74 @@
+import time
+import asyncio
+import lsst.daf.persistence as dafPersist
+import logging
+from lsst.rapid.analysis import BestEffortIsr
+
+STD_TIMEOUT = 10  # seconds
+
+logging.basicConfig()
+# Make matplotlib less chatty
+logging.getLogger("matplotlib").setLevel(logging.WARNING)
+logger = logging.getLogger(__name__)
+
+
+async def get_image(
```

Comment:
I think we need to make this method part of a class... Consider creating a `ImageGetter` class that contains a `get_image` method.

```
class ImageGetter:

    def __init__(self, dataset="quickLookExp", datapath="/project/shared/auxTel/", timeout=10, runBestEffortIsr=True, loop_time=0.1, log=None):
        ...
        
    def get_image(self, data_id):
        ...
```

Note that I am passing a "log" to the class... This means you can handle logging more appropriately than with the way it is doing here.